### PR TITLE
Asp deployment

### PR DESCRIPTION
**Pull request by scottyhq**

### CODE REVIEW

File: deployment/1_create_iam_user.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#sed -e "s/CHANGE_ACCOUNT_NUMBER/$ACCOUNT/g" ./templates/template-eksctl-permissions.json > eksctl-permissions.json
+#aws iam create-policy --policy-name eksctl-permissions --policy-document file://eksctl-permissions.json
+#aws iam attach-user-policy --policy-arn arn:aws:iam::$ACCOUNT:policy/eksctl-permissions --user-name circle-ci-deploy
+
+
+#2) CUSTOMIZE configuration and permissions
+#KEY=CHANGE
+#SECRET_KEY=CHANGE
+#aws configure set aws_access_key_id ${KEY}
+#aws configure set aws_secret_access_key ${SECRET_KEY}
```
Comment (jhamman) [inline deployment/1_create_iam_user.sh:20]: Curious why all of these are commented out? Would a new user need to run these?

File: deployment/4_configure_kubernetes.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# these commands are taken from zero2jupyterhub docs
+EMAIL=scottyh@uw.edu
```
Comment (jhamman) [inline deployment/4_configure_kubernetes.sh]: we can probably remove this.

**Comment by jhamman** [comment]
Looks great @scottyhq - just a few comments....

File: deployment/1_create_iam_user.sh (hunk 1 above)
Comment (scottyhq) [reply deployment/1_create_iam_user.sh:20]: yep - for a brand new account or setting up aws cli for the first time, `aws configure` will write the configuration files to `~/.aws/credentials` and `~/.aws/config` for future commands

File: deployment/4_configure_kubernetes.sh (hunk 2 above)
Comment (scottyhq) [reply deployment/4_configure_kubernetes.sh]: ok

**Pull request merged**